ui/analyze_window.cpp

```diff
@@ -642,14 +650,12 @@ absl::Status AnalyzeDialog::Pm4Replay(Dive::DeviceManager &device_manager,
     else if (!m_dump_pm4_enabled && m_gpu_time_enabled)
     {
         SetReplayButton("Replaying with gpu_time enabled...", false);
-        SetReplayDownloadDir();
```

Comment:
Is this still needed if we want to download a csv that contains the gpu time info?

https://github.com/google/dive/pull/338#discussion_r2304798460

Reply:
Moved to AnalyzeDialog::OnReplay() so that the the download directory is set when performing gpu time, perf counter, or dump pm4 replay